src/scss/_utility.scss

```diff
@@ -33,14 +33,73 @@
 
 
 //tease 2.0
-.bg-artemis-dark {
-  background-color: $artemis-dark !important;
+
+/* Background Styles */
+.bg-dark {
```

Comment:
You can define your colors and add them to the Bootstrap theme. This way you can access your custom colors in the same way as the bootstrap colorswithout explicitly defining them, e.g.` text-primary`, `text-tile-priority`. 
Let me show you how to do this.